service/title_formatter.py

```diff
@@ -31,6 +35,20 @@ def format_display_json(title_json):
     return title
 
 
+def _format_a1_notes(title_number):
+    # grabs official copy data, extracts the a register sub register and then takes the notes
+    official_copy_data = api_client.get_official_copy_data(title_number)
+    sub_registers = official_copy_data.get('official_copy_data', {}).get('sub_registers')
```

Comment:
Yes, i double checked this one!  If I had written get('sub_registers', {} (or [])) it would have returned something empty or with length == 0 instead of none, i decided none was better.